mrtt-ui/src/components/ManagementStatusAndEffectiveness/ManagementStatusAndEffectivenessForm.js

```diff
@@ -0,0 +1,375 @@
+import { useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
+import axios from 'axios'
+import { toast } from 'react-toastify'
+import { Controller, useForm } from 'react-hook-form'
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns'
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
+import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker'
+import { Box, MenuItem, Stack, TextField } from '@mui/material'
+
+import { Form, FormPageHeader, FormQuestionDiv, StickyFormLabel } from '../../styles/forms'
+import { ContentWrapper } from '../../styles/containers'
+import { managementStatusAndEffectiveness as questions } from '../../data/questions'
+import CheckboxGroupWithLabelAndController from '../CheckboxGroupWithLabelAndController'
+import { multiselectWithOtherValidationNoMinimum } from '../../validation/multiSelectWithOther'
+import LoadingIndicator from '../LoadingIndicator'
+import QuestionNav from '../QuestionNav'
+import useSiteInfo from '../../library/useSiteInfo'
+import useInitializeQuestionMappedForm from '../../library/useInitializeQuestionMappedForm'
+import { ErrorText, PageSubtitle, PageTitle } from '../../styles/typography'
+import language from '../../language'
+import { mapDataForApi } from '../../library/mapDataForApi'
+import { questionMapping } from '../../data/questionMapping'
+
+const ManagementStatusAndEffectivenessForm = () => {
+  const { site_name } = useSiteInfo()
+  const validationSchema = yup.object({
+    dateOfAssessment: yup.string(),
+    stakeholderManagement: multiselectWithOtherValidationNoMinimum,
+    stakeholderInfluence: yup.string(),
+    managementStatusChanges: yup.string(),
+    currentManagementStatus: yup.string(),
+    managementLaws: yup.string(),
+    nameOfFormalManagementArea: yup.string(),
+    projectStatusChange: yup.string(),
+    currentProtectionStatus: yup.string(),
+    financeForCiteManagement: yup.string(),
+    sufficientFunds: yup.string(),
+    resourcesToEnforceRegulations: yup.string(),
+    equitableSharingOfSiteBenefits: yup.string(),
+    climageChangeAdaptation: yup.string()
+  })
+  const reactHookFormInstance = useForm({
+    defaultValues: {
+      stakeholderManagement: { selectedValues: [], otherValue: undefined }
```

Comment:
why not default all the form values? I like it because it is a kind of explicit self documentation.

Reply:
I'm mostly concerned with any that are arrays, as we have lots of code that doesn't account for these being null. Other default vals are set in the controller, such as empty strings so we don't get errors